filecache_test.go

```diff
@@ -264,5 +264,13 @@ var _ = Describe("Filecache", func() {
 
 			Expect(fname).To(HaveSuffix(".asdf"))
 		})
+
+		It("prepends a directory to the file path with its name being the first byte of the FNV32 hash of the file name", func() {
+			fname := cache.GetFileName("document.pdf")
+
+			dir := filepath.Dir(fname)
+
+			Expect(dir).To(Equal("c9"))
```

Comment:
Ideal test would then be to hash a different value and see that the directory is different :)

Reply:
That's a good point! :)